codegen/tests/unit/responses/iam-list-access-keys.xml

```diff
@@ -0,0 +1,20 @@
+<ListAccessKeysResponse xmlns="https://iam.amazonaws.com/doc/2010-05-08/">
```

Comment:
This and the other files were added because (per the discussion earlier in the PR), the botocore versions of the responses are broken, so we'll use our own corrected ones until they're fixed upstream.